Source/MediaInfo/Export/Export_EbuCore.cpp

```diff
@@ -1544,12 +1544,27 @@ Ztring Export_EbuCore::Transform(MediaInfo_Internal &MI, version Version, acquis
     Node* Node_Format=new Node("ebucore:format");
 
     //Use external metadata
-    if (UseExternalMetaData && !ExternalMetadata(MI.Get(Stream_General, 0, General_FileName), ExternalMetadataValues, ExternalMetaDataConfig, Node_CoreMetadata, Node_Format))
+    if (UseExternalMetaData)
     {
-        delete Node_CoreMain;
-        delete Node_Format;
+        Ztring FileName;
+        if (!MI.Get(Stream_General, 0, General_FileName).empty())
+            FileName=MI.Get(Stream_General, 0, General_FileName);
+        if (!MI.Get(Stream_General, 0, General_FileExtension).empty())
+            FileName+=__T('.')+MI.Get(Stream_General, 0, General_FileExtension);
+        if (FileName.empty())
+        {
+            MediaInfoLib::Config.Log_Send(0xC0, 0xFF, 0, "File name not available");

```

Comment:
"File name not found in external metadata file"

Reply:
PR updated